docs/cadt_rpc_api.md

```diff
@@ -95,67 +114,135 @@ GET Options: None
+    "orgUid":"77641db780adc6c74f1ff357804e26a799e4a09157f426aac588963a39bdb2d9",
+    "orgHash": "0x14d8ea0f809c73c649827837cada5ec4d931153839383008a28c59fd1de86d2e",
+    "name":"Org Test",
+    "icon":"https://www.chia.net/wp-content/uploads/2023/01/chia-logo-dark.svg",
+    "isHome":true,
+    "subscribed":true,
+    "synced": true,
+    "fileStoreSubscribed": "0",
+    "registryId": "xfy7oofvb31bg07stafbqxcug7mmmjzxg0gi8r1nwkv63u3pxwy85s5xpgs204bk",
+    "registryHash": "0x34c4671f721ff0132b4eb80a8e0d46ffb446ec8f03ed87368adcd29415cdbac4",
+    "sync_remaining": 0
+    },
+  "37651db8203d4c6c71f1ff357804eg6ag99eaa0k15f426aac5c896ga39bddji2":{
+    "orgUid":"37651db8203d4c6c71f1ff357804eg6ag99eaa0k15f426aac5c896ga39bddji2",
+    "orgHash": "0xb238e414a8df2ff5f1d1a67ca3db35c40bcb021624d5f34f9b539c0fa800272f",
+    "name":"Number 2 Org Test",
+    "icon":"https://www.chia.net/wp-content/uploads/2023/01/chia-logo-light.svg",
+    "isHome":true,
+    "subscribed":true,
+    "synced": false,
+    "fileStoreSubscribed": "0",
+    "registryId": "aigcyhuhhe4wxp2owt6j7cezr5dqpluh7g1ybtopkn2cvxz1vjjwnw146xhzuq3i",
+    "registryHash": "0x38vhoibwjnchsw0re23dsxnb7cr64dg7v717sfqkfcrs94e8z03munmnzk8jpes9",
+    "sync_remaining": 0
+  }
 }
+
 ```
 
 ---
 
 POST Options:
 
-| Key  |  Type  | Description                                                 |
-| :--: | :----: | :---------------------------------------------------------- |
-| name | String | (Required) Name of the organization to be created           |
-| icon | String | (Required) URL of the icon to be used for this organization |
+|  Key   |   Type   |                         Description                          |
+|:------:|:--------:|:------------------------------------------------------------:|
+|  name  |  String  |      (Required) Name of the organization to be created       |
+|  icon  |  String  | (Required) URL of the icon to be used for this organization  |
 
 ### POST Examples
 
 #### Create an organization
 
-```json
-// Request
+Request
+```sh
 curl --location -g --request POST 'localhost:31310/v1/organizations/create' \
      --header 'Content-Type: application/json' \
      --data-raw '{
         "name": "Sample Org",
         "icon": "https://www.chia.net/wp-content/uploads/2023/01/chia-logo-dark.svg"
 }'
+```
 
-// Response
+Response (after up 30 minutes)
+```json
 {
   "message":"New organization created successfully.",
-  "orgId":"d84ab5fa679726e988b31ecc8ecff0ba8d001e9d65f1529d794fa39d32a5455e"
+  "orgUid":"d84ab5fa679726e988b31ecc8ecff0ba8d001e9d65f1529d794fa39d32a5455e"
 }
 ```
 
 ---
 
-PUT Options: None
+PUT Options: 
+
+|  Key   |  Type   |                     Description                      |
+|:------:|:-------:|:----------------------------------------------------:|
+| orgUid | String  | (Required) OrgUid of the home organization to import |
 
 ### PUT Examples
 
-#### Subscribe to an organization
+#### Import a home organization that datalayer is subscribed to
```

Comment:
Feel free to wordsmith a bit, but want it approximately "Import a home organization. This is typically used when the CADT server has been put onto a new machine."

Reply:
noted